packages/flutter/lib/src/services/text_formatter.dart

```diff
@@ -120,6 +122,68 @@ class BlacklistingTextInputFormatter extends TextInputFormatter {
       = new BlacklistingTextInputFormatter(new RegExp(r'\n'));
 }
 
+/// A [TextInputFormatter] that prevents the insertion of more characters (i.e.
+/// Unicode runes) than allowed.
```

Comment:
runes => scalar values

Also I don't think "i.e." is really right here. Characters aren't Unicode scalar values. I think you want to say something like "of more Unicode scalar values".

Reply:
or "of more characters (currently defined as Unicode scalar values)".